### PR TITLE
PAR-1659 document count data clean

### PR DESCRIPTION
**Pull request by jon-edery**
Screen Shot 2020-07-29 at 16 43 25

### CODE REVIEW

File: web/modules/custom/par_forms/src/Plugin/ParForm/ParPartnershipDocuments.php [hunk 1]
```diff
@@ -93,17 +106,20 @@ public function getElements($form = [], $cardinality = 1) {
...
-          ->getFlow()
-          ->getNextLink('advice', [], [], TRUE);
+        $add_advice_list_link = $this->getFlowNegotiator()->getFlow()->getFlowLink('advice', 'See all Advice');
       } catch (ParFlowException $e) {
 
       }
       if (isset($add_advice_list_link) && $add_advice_list_link instanceof Link) {
+        $form['details']['advice']['document_count'] = [
+          '#type' => 'html_tag',
+          '#tag' => 'div',
```
Comment (kalpaitch) [inline web/modules/custom/par_forms/src/Plugin/ParForm/ParPartnershipDocuments.php:116]: I noticed there was some different formatting here, needs a <p> tag like the one above.

Comment (jon-edery) [reply web/modules/custom/par_forms/src/Plugin/ParForm/ParPartnershipDocuments.php:116]: yea good spot that should be a p tag

**Comment by kalpaitch** [comment]
Let's wait till we've got the next release approved before merging.

**Comment by jon-edery** [comment]
++

**Pull request merged**